client/components/spinner/docs/example.jsx

```diff
@@ -18,9 +18,22 @@ module.exports = React.createClass( {
 		return (
 			<div>
 				<p>
-					<strong>Please exercise caution in deciding to use a spinner in your component.</strong> A lone spinner is a poor user-experience and conveys little context to what the user should expect from the page. Refer to <a href="/devdocs/docs/reactivity.md">the <em>Reactivity and Loading States</em> guide</a> for more information on building fast interfaces and making the most of data already available to use.
+					<strong>Please exercise caution in deciding to use a spinner in your component.</strong>&nbsp;
+					A lone spinner is a poor user-experience and conveys little context to what the user should expect from the page.&nbsp;
+					Refer to <a href="/devdocs/docs/reactivity.md">the <em>Reactivity and Loading States</em> guide</a>&nbsp;
+					for more information on building fast interfaces and making the most of data already available to use.
 				</p>
+
+				<strong>Spinner</strong>
 				<Spinner />
+
+				<br />
+				<strong>counterClockwise: true</strong>
+				<Spinner counterClockwise={ true } />
```

Comment:
We don't need to add the explicit `={ true }`; doing `<Spinner counterClockwise />` is enough. Or is the former preferred?